composer.json

```diff
@@ -14,7 +14,8 @@
     "require": {
         "php": "^5.5.6|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0"
+        "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
+        "sonata-project/admin-bundle": "^3.6.0"
```

Comment:
The ContentBundle's form type relies on some behaviour that's introduced in SonataAdminBundle 3.6.0. If we don't specify this, we seem to also support SonataAdminBundle 3.1.0 (the lowest support version from the dependencies). See also the failing previous builds on this branch.